FeedAPIChallenge/Feed API/FeedImageItemMapper.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  FeedImageItemMapper.swift
+//  FeedAPIChallenge
+//
+//  Created by Guilherme Sol on 11/09/2021.
+//  Copyright © 2021 Essential Developer Ltd. All rights reserved.
+//
+
+import Foundation
+
+internal class FeedImageItemMapper {
+	
+	private static var OK_200:Int{ 200 }
+	
+	private struct Root: Decodable {
+		var items: [FeedImageItem]
+		var feedItems: [FeedImage] {
+			items.compactMap({ $0.feedImage })
```

Comment:
If `$0.feedImage` returns nil, we are supposed to complete with `.invalidData` instead of ignoring that particular item using `.compactMap`. 

In our case that might happen because of an invalid `URL` - so we should deal with invalid cases during mapping.

Please replace `compactMap` with `map`:

```suggestion
			items.map({ $0.feedImage })
```